osf/migrations/0171_add_registration_files_count.py

```diff
@@ -19,23 +16,25 @@ def add_registration_files_count(state, *args, **kwargs):
     relationship for speed purposes in this migration.  If this model changes significantly,
     this migration may have to be modified in the future so it runs on an empty db.
     """
-    registrations = Registration.objects.filter(is_deleted=False).filter(
-        files__type='osf.osfstoragefile',
-        files__deleted_on__isnull=True
-    ).annotate(
-        annotated_file_count=Count('files')
-    )
-    progress_bar = tqdm(total=registrations.count())
+    Registration = state.get_model('osf', 'Registration')
+    registrations = Registration.objects.filter(is_deleted=False, files_count__isnull=True)
```

Comment:
So this PR is meant to just create and populate the 'deleted' field, and not begin utilization as of yet, because the data might not be populated (depending on how long the data load takes). The next PR to this will remove 'is_deleted' and change utilization.

'files_count' is just on the registration model.